mapie/tests/test_regression.py

```diff
@@ -463,7 +509,7 @@ def test_linear_data_confidence_interval(strategy: str) -> None:
     """
     mapie = MapieRegressor(**STRATEGIES[strategy])
     mapie.fit(X_toy, y_toy)
-    y_pred, y_pis = mapie.predict(X_toy, alpha=0.2)
+    y_pred, y_pis = mapie.predict(X_toy, alpha=0.5)
```

Comment:
I'm confused about why we change the level of `alpha`.

Reply:
Because of the UPD of the quantile formula.
- we have a toy dataset with 6 exemples
- we divide it into two data sets with split method (50%) thus n_calib=3
- when we use alpha=0.2, we need at least 1/(0.2)=5 calibration data (but here we have 3)
- I propose to use alpha=0.5, we need at least 1/(0.5)=2 calibration data